oasis/deployments.md

```diff
@@ -166,6 +166,47 @@ and automatically accepted by most web browsers. The certificate used on port
 a self-signed certificate is recommended.
 {% endhint %}
 
+## How to create a private network deployment
+
+Microsoft Azure offers a feature called
+[Azure Private Link](https://docs.microsoft.com/en-us/azure/private-link){:target="_blank"}
+that allows you to limit communication between different Azure servers and
+services to Microsoft's backbone network without exposure to the public internet.
+It can lower network latency and increase security.
+
+If you want to connect an ArangoDB Oasis deployment running on Azure with other
+services you run on Azure using such a tunnel, then
+[create a regular deployment](#how-to-create-a-new-deployment) and change it
+to a private network deployment afterwards.
+
+The deployment will not be reachable from public internet anymore, other than via
+the Oasis dashboard to administrate it. To revert to a public deployment, please
+contact support via _Request help_ in the help menu.
+
+1. **View** the deployment you want to change.
+2. On the **Overview** tab, click the **Edit** button with an ellipsis (`…`)
+   icon. If you see a pencil icon and no menu opens, then you are either on the
+   free-to-try tier or the selected deployment is not eligible. The private
+   network option in only available on the professional tier, for OneShard and
+   Sharded deployments running on Azure.
+3. Click **Change to private network** in the menu
+   ![Oasis Deployment Private Network Menu](images/oasis-deployment-private-network-menu.png)
+4. A dialog opens for the configuration. Read the explanation and click **Next**.
```

Comment:
Maybe we can call this a wizard instead of a dialog since it takes the users' through different steps?

Reply:
Second that. Would rephrase altogether as "In the Configuration wizard, click Next to enter your configuration details."